config.yaml

```diff
@@ -36,7 +36,7 @@ modelsRuntime:
 #          tag: latest
       persistence:
```

Comment:
Considering to move the whole section to hardcoded default value (in values.yaml or in CDK provided values), since we will rely on static provisioning for a long time